data/mods/No_Hope/Mapgen/mil_base_z0.json

```diff
@@ -1036,7 +1037,7 @@
         "|T+..S|[.............=..#h.|ss  ______________________________________________________,,,,b ,,,x",
         "--------ww-------|-+-|Y.#..wssssssss__________________________________________________,,,,b,,, x",
         "sssssssssss~|T+.S|...|O...$|ssssssss            sssssss            sXsssXXXXXXXXXs    ,,, b,,,,x",
-        "ssssssssssss|--.c|...|------------ss         sssssssssssss         sXs sGsssXsssGs     ,,,b,,, x",
+        "ssssssssssss|--.c|...|------------ss         sssssssssssss         sXspsGsssXsssGs     ,,,b,,, x",
```

Comment:
This "p" (4th character in) is present in the standard version of the map and I don't see why it would be deliberately changed.